datasets/prepare_all.sh

```diff
@@ -6,18 +6,26 @@ python3 ${SCRIPT_DIR}/gsm8k/prepare.py --split_name validation
 python3 ${SCRIPT_DIR}/gsm8k/prepare.py --split_name test
 # train + validation - should be used for the final run only
 python3 ${SCRIPT_DIR}/gsm8k/prepare.py --split_name train_full
+echo "Preparing gsm-plus"
```

Comment:
Please include all of the new datasets in here https://github.com/Kipok/NeMo-Skills/blob/main/tests/test_datasets.py#L63